haystack/pipelines/utils.py

```diff
@@ -250,31 +254,37 @@ def print_eval_report(
         n_wrong_examples=n_wrong_examples,
         document_scope=document_scope,
         answer_scope=answer_scope,
+        fields=wrong_examples_fields,
+        max_chars=max_characters_per_field,
     )
 
     print(f"{pipeline_overview}\n" f"{wrong_examples_report}")
 
 
-def _format_document_answer(document_or_answer: dict):
-    return "\n \t".join(f"{name}: {value}" for name, value in document_or_answer.items())
+def _format_document_answer(document_or_answer: dict, max_chars: int = None, field_filter: List[str] = None):
+    if field_filter is None or len(field_filter) == 0:
+        field_filter = document_or_answer.keys()  # type: ignore
+    return "\n \t".join(f"{name}: {str(value)[:max_chars]}" for name, value in document_or_answer.items() if name in field_filter)  # type: ignore
```

Comment:
It would be awesome (but not mandatory) to have trailing `...` when we cut. But I don't want to stress this PR too much. It's already a huge improvement.